relayer-cli/src/commands/tx/connection.rs

```diff
@@ -58,7 +58,7 @@ impl Runnable for TxRawConnInitCmd {
             self,
             |chains: ChainHandlePair| {
                 Connection {
-                    delay_period: Duration::from_secs(0),
```

Comment:
Maybe consider introducing a DEFAULT_PACKET_DELAY const in the codebase? Similar to how we have a [`MAX_PACKET_DELAY`](https://github.com/informalsystems/ibc-rs/blob/c05f9c16231216bcd3ab2e7ca07626eafc4c9d3c/relayer/src/connection.rs#L27).

Reply:
I have instead defined `ZERO_DURATION`.